cmd/lsjson/lsjson.go

```diff
@@ -17,26 +18,29 @@ import (
 )
 
 var (
-	recurse   bool
-	showHash  bool
-	noModTime bool
+	recurse       bool
+	showHash      bool
+	showEncrypted bool
+	noModTime     bool
 )
 
 func init() {
 	cmd.Root.AddCommand(commandDefintion)
 	commandDefintion.Flags().BoolVarP(&recurse, "recursive", "R", false, "Recurse into the listing.")
 	commandDefintion.Flags().BoolVarP(&showHash, "hash", "", false, "Include hashes in the output (may take longer).")
 	commandDefintion.Flags().BoolVarP(&noModTime, "no-modtime", "", false, "Don't read the modification time (can speed things up).")
+	commandDefintion.Flags().BoolVarP(&showEncrypted, "encrypted", "M", false, "Shows the names encrypt.")
 }
 
 // lsJSON in the struct which gets marshalled for each line
 type lsJSON struct {
-	Path    string
-	Name    string
-	Size    int64
-	ModTime Timestamp //`json:",omitempty"`
-	IsDir   bool
-	Hashes  map[string]string `json:",omitempty"`
+	Path      string
+	Name      string
+	Encrypted string
```

Comment:
This should probably have a

    `json:",omitempty"`

tag like the other optional fields so they don't appear in the JSON if the flag isn't used.